ports/bare-arm/mpconfigport.h

```diff
@@ -37,6 +37,9 @@
 // Python internal features
 #define MICROPY_ERROR_REPORTING                 (MICROPY_ERROR_REPORTING_NONE)
 
+// Fine control over Python builtins, classes, modules, etc.
+#define MICROPY_PY_SYS                          (0)
```

Comment:
What's the code size cost if we make the `bare-arm` config the same as `ports/minimal` here?

(I guess what I'm hinting it as whether "minimal" can itself be a feature level?)

Reply:
> What's the code size cost if we make the `bare-arm` config the same as `ports/minimal` here?

It's +412 bytes.

But I kind of wanted to have a build without `sys` (bare-arm) and one with "minimal" `sys` (minimal port), if only to test that MicroPython can build with such configuration.

Reply:
I see, makes sense!